utils/spacewalk-common-channels.ini

```diff
@@ -1593,6 +1593,15 @@ checksum = sha256
 base_channels = ubuntu-16.04-pool-amd64
 repo_url = http://archive.ubuntu.com/ubuntu/dists/xenial-updates/universe/binary-amd64/
 
+[ubuntu-1604-amd64-universe-security]
+label    = ubuntu-1604-amd64-universe-security
+name     = Ubuntu 16.04 LTS AMD64 Universe Security Updates
+archs    = amd64-deb
+repo_type = deb
+checksum = sha256
+base_channels = ubuntu-16.04-pool-amd64
+repo_url = http://archive.ubuntu.com/ubuntu/dists/xenial-security/universe/binary-amd64/
```

Comment:
@lucidd and I are currently discussing if we should change the URL to the security repos to security.ubuntu.com as hostname. They may get the updates earlier than archive.ubuntu.com. Not sure if you see this as relevant.

Reply:
ACK. For now let's merge it as it is. It's easy to adjust later if needed.